src/public/navigation.ts

```diff
@@ -0,0 +1,16 @@
+import { ensureInitialized, sendMessageRequestToParent } from '../internal/internalAPIs';
+
+/**
+ * Namespace to interact with the navigation specific part of the SDK.
+ */
+export namespace navigation {
```

Comment:
@sachingupta, how do you feel about this namespace? Today we have navigation APIs at the top level (e.g. navigateBack) so is the idea that we would move them into here at some point? Or should we just put this API at the top level since it's part of the "core" functionality?

Reply:
I spend some time thinking about place for returnFocus API. Lets go with putting return focus at top level so developer call it using microsoftTeams.returnFocus because

	
	Many other navigation APIs are already at top level - we can't move those under namespace without breaking changes.
	
	
	As Yuri also mentioned we can treat all navigation APIs as core APIs.
	
For better code organization Lets move navigation related APIs in a separate file - called navigationAPIs.ts without using namespace.